README.md

```diff
@@ -270,7 +270,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Motorola Solutions](https://motorolasolutions.wd5.myworkdayjobs.com/en-US/Careers/job/West-Valley-City-UT/Software-Engineer--2023-Summer-Internship-_R32052) | West Valley City, UT | Software Engineer (2023 Summer Internship) |
 | [Ansys](https://careers.ansys.com/job/Canonsburg-Summer-2023-Intern-Ansys-Discovery-Post-processing-%28BachelorsMasters%29-PA-15317/934118500) | Canonsburg, PA | Summer 2023 Software Development Intern - Ansys Discovery Post-processing (BS or MS) |
 | [Silicon Labs](https://jobs.jobvite.com/silabs/search?q=&l=&c=University%20Recruiting) | Austin, TX | [Embedded Software Engineering Intern (Austin, TX; or Boston, MA)](https://jobs.jobvite.com/silabs/job/oZgJkfw2), [Software Engineering Intern](https://jobs.jobvite.com/silabs/job/oV6LkfwQ) |
-| ASML | Wilton, CT | **🔒 Closed 🔒** Intern, Software Engineer (Summer 2023) |
+| [ASML](https://www.asml.com/en/careers/internships?page=1&facets=vacancycountry%253dUS&sortBy=date) | Wilton, CT; San Jose, CA; | [Software Test Engineer intern](https://www.asml.com/en/careers/find-your-job/5/3/0/software-test-engineer-intern-req53028), [Devops Intern](https://www.asml.com/en/careers/find-your-job/5/2/5/devops-intern-req52528?ppc=JBP-181) (additional on website) |
```

Comment:
Are you sure these internships are closed?

Both Software Test Engineer Intern and Devops Intern point me to the application page. What do you see when you try those links?

Reply:
Umm, sorry if I missed something, but I believe ASML is currently marked closed in upstream, and I'm reopening it by adding links to Devops, Software Test Engineer intern, and general- the other way around. 

The posting I have closed is Lucid software since those postings aren't up anymore. Let me know if that clarified it.

Reply:
Oh shoot, I can't read lol

Reply:
haha all good